tomcat-main/src/main/resources/defaults/base-collectionobject.xml

```diff
@@ -194,6 +194,12 @@
 				<field id="inscriptionDescriptionInterpretation" />
 			</repeat>
 		</section>
+		<repeat id="annotationGroupList/annotationGroup">
+			<field id="annotationType" autocomplete="true" ui-type="enum" />
+			<field id="annotationNote" />
+			<field id="annotationDate" datatype="date" />
+			<field id="annotationAuthor" autocomplete="true" />
+		</repeat>
```

Comment:
This is going to put these fields in the `collectionobjects_common` namespace, which is a change from where they used to be, in the `collectionobjects_annotation` namespace. We should avoid doing this, because it forces everyone with existing data to migrate the data, and change any reports and integrations that expect to find the fields in a particular place. It forces us to make a major release. Basically it's a pain for everyone. Instead, let's keep these in the `collectionobjects_annotation` namespace. It's fine to have them defined in this file, but you need to copy over the `<services-record-path id="annotation">` element from `annotation-collectionobject.xml`, and add the `section="annotation"` attribute back on to the `repeat` and `field` elements.

Reply:
@ray-lee I just pushed a commit to add back in the `collectionobjects_annotation` namespace to this and `cspace-ui.js`. I did a quick test against core and was able to save but haven't checked against other tenants yet.